fs/ggml/gguf.go

```diff
@@ -628,7 +712,7 @@ func ggufWriteTensorInfo(ws io.WriteSeeker, t Tensor) error {
 	}
 
 	for i := range len(t.Shape) {
-		if err := binary.Write(ws, binary.LittleEndian, t.Shape[len(t.Shape)-i-1]); err != nil {
```

Comment:
This writes the shape in reverse which is necessary for ggml. Is that no longer the case?